irohad/main/application.cpp

```diff
@@ -952,7 +913,7 @@ Irohad::RunResult Irohad::initWsvRestorer() {
  * Run iroha daemon
  */
 Irohad::RunResult Irohad::run() {
-  ordering_gate->onProposal().subscribe(
+  ordering_init->subscribe(
```

Comment:
why ordering_init instead of ordering_gate?

Reply:
Because subscription logic was decoupled from OrderingGate, and moved to initialization. Probably it will be possible to move more code from ordering init to ordering classes, but subscription will be in application.cpp then.